other-docs/guides/getting-help-with-altis.md

```diff
@@ -1,50 +1,91 @@
+
+**Type**
+
+The type of support ticket, of which there are currently 4. Question, Incident, Problem, and Task. We'll go into more detail about which these categories below, but most tickets will generally fall into one of these four. 
+
+**Priority**
+
+You can change the priority of a support issue, if you need to. In most cases you shouldn't need to change this, but feel free to let us know if something is a priority.
+
+**Subject**
+
+Short line describing the support request.
+
+**Description**
+
+The body of the support request.
+
+**CCs**
+
+You can CC additional relevant parties to the support request if desired. They will recieve carbon copies of the correspondance.
+
+## Support Ticket Types
+
+We break down support into different categories. Each has their own language and processes, but they should be somewhat intuitive. Let's go over the types.
+
+![Altis Dashboard Create Support Ticket page, Question selected in Type dropdown menu](./assets/altis-support-type.png)
 
 We currently have four types of support. Below is a description of each and how they are different from each other.
 
 ### Incident
 
-Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if something isn’t getting the expected result, this would be an Incident, and requires investigation.
+Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if there is an issue which is causing an immediate business impact, this would be an Incident, and you can make a support request to help investigate.
 
-From the support side, Incidents get a rapid response. If the site is down, this is an Incident, with a priority of ‘Urgent’. Alarm bells will ring, and we’ll put out the fires to get things running again by any means necessary.
+From the support team side, Incidents get a rapid response. Alarm bells will ring, and we’ll put out the fires to get things running again by any means necessary.
 
-Incidents focus on immediate attention, with an emphasis on immediate remediation. Remediating an Incident can take the form of temporary patches, which brings us to the next support type.
+If the site is down for example, select the 'Incident' type, with a priority of ‘Urgent’.
 
 ### Problem
 
-Problems are the cause of, or potential cause of Incidents. Incidents revolve around something unexpected. If a site is down, that’s an Incident, but the Problem is a known cause of an issue you wish to track, and coordinate help with. Problems typically take longer to resolve, as an appropriate solution is thought out and worked on.
+Problems are the cause of, or can be the potential cause of future Incidents. If you're encountering a problem which could cause potential Incidents, let us know!
```

Comment:
This is way more concise and clear and doesn't use the definition of an Incident as part of the description of what a Problem is. 👍